ci/e2e-tests/suite-setup.sh

```diff
@@ -80,8 +80,13 @@ az iot dps linked-hub create \
 # `az iot dps linked-hub create` deletes the tags on the DPS that
 # were set by `az iot dps create` for some unknown reason, so we
 # need to tag it again.
+#
+# A bug in the latest az CLI causes az resource tag to use an API
+# version not supported by the cloud. For now, we fill manually
+# specify the API version.
```

Comment:
Add the link to the bug too ( https://github.com/Azure/azure-cli/issues/20263 ) so we can check on it later and remove this.

Also, this is the only command that needs this workaround?